eng/scripts/Language-Settings.ps1

```diff
@@ -47,10 +47,9 @@ function Publish-c-GithubIODocs ($DocLocation, $PublicArtifactLocation)
     # Those loops are left over from previous versions of this script which were
     # used to publish multiple docs packages in a single invocation.
     $pkgInfo = Get-Content $DocLocation/package-info.json | ConvertFrom-Json
-    $releaseTag = RetrieveReleaseTag "C" $PublicArtifactLocation
+    $releaseTag = RetrieveReleaseTag -artifactLocation $PublicArtifactLocation
```

Comment:
I am not very familiar with this script, I am going to assume this is somehow doing the right thing now :)

Reply:
This file was missed in a SDK-wide update of release tag retrieval.